bridge/bridgeapp/forms.py

```diff
@@ -1,38 +1,49 @@
 from django import forms
 from bridgeapp.models import Category
 
+# global variable with assigned loop to create a dictionary of category.id : category.type key:value pairs
 CAT_CHOICES = [(f"{category.id}", f"{category.type}")
                for category in Category.objects.all()]
 
 
 class ResponseForm(forms.Form):
     """Form used for Responses to Threads"""
+    # simple form with a single CharField
     body = forms.CharField(
         required=True,
         label='',
         widget=forms.Textarea(attrs={
             'rows': '3',
             'cols': '50',
+            # causes cursor to focus on text input area upon page load
             'autofocus': True,
+            # inserts placeholder text into text input area as prompt for user
             'placeholder': 'Write a response here...'
         }))
 
 
 class ThreadForm(forms.Form):
     """Form used for creation of Threads"""
     body = forms.CharField(
+        # requires entry in this field before form can be submitted
```

Comment:
"Required field" says everything you say here.
Again a `required` attribute really don't need a comment. If anything I would reference it in one comment for the entire `CharField`, such as:
```
# a <textarea> field for Thread.body that uses place holder as label and is autofocused
```